app/assets/stylesheets/BlackOnWhite.scss

```diff
@@ -1,2 +1,22 @@
 @import "defaults";
 @import "mushroom_observer";
+
```

Comment:
Removing borders on boxes in black and white theme.  Couldn't figure out how to do this through the bootstrap sass package.